app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -236,19 +244,20 @@ private fun PocketStoriesComposablesPreview() {
                 PocketStories(
                     stories = getFakePocketStories(8),
                     client = FakeClient(),
-                    onExternalLinkClicked = { }
-                )
+                    contentPadding = 0.dp
+                ) { }
                 Spacer(Modifier.height(10.dp))
 
                 PocketStoriesCategories(
                     "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor".split(" ").map {
                         PocketRecommendedStoriesCategory(it)
                     },
-                    emptyList()
-                ) { }
+                    emptyList(),
+                    { }
+                )
                 Spacer(Modifier.height(10.dp))
 
-                PoweredByPocketHeader { }
+                PoweredByPocketHeader({ })
```

Comment:
Could we set `onExternalLinkClicked: (String) -> Unit = {}` so we can avoid the `({ })` here?

Reply:
The click listener is a required parameter which I'd prefer to keep as such.
Although the code here and it's usage is straightforward and we'd know now to actually pass a listener and not rely on the default I feel it's important to keep the composable code exposed to clients as clean and tight as possible while for the private code and here previews we can maybe a bit uglier setup.